### PR TITLE
O3-1747: Updated Drug Order Form UI for desktop and tablet as per latest designs

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary
This PR brings the latest Drug Order form UI for desktop and tablet views as per the latest design changes by Ciaran.
Design: 

**Tablet**: [here](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/62c6bf3535e7641646bf9421)
**Desktop**: [here](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/62c6bf37c880131a4ea0d64a)

## Screenshots
https://user-images.githubusercontent.com/51502471/212709025-11f9974a-651f-43a8-b00b-263aa09bbdb8.mov

## Related Issue
https://issues.openmrs.org/browse/O3-1747

## Other
None

### CODE REVIEW

File: packages/esm-patient-medications-app/src/root.scss [hunk 1]
```diff
@@ -61,3 +61,7 @@
   overflow: hidden;
   white-space: nowrap;
 }
+
+.text02 { 
+  color: $text-02;
+}
```
Comment (hadijahkyampeire) [inline packages/esm-patient-medications-app/src/root.scss:67]: nit: Add a line at the end

File: packages/esm-patient-medications-app/src/order-basket/medication-order-form.scss [hunk 2]
```diff
@@ -99,84 +126,41 @@
...
-}
-
-.dosingWrapper {
-  display: flex;
-}
-
-.dosingWrapper :global(.bx--combo-box) {
-  width: 75% !important;
-}
+}
```
Comment (hadijahkyampeire) [inline packages/esm-patient-medications-app/src/order-basket/medication-order-form.scss:166]: same here, empty line.

**Review by hadijahkyampeire — APPROVED** [review]
Very nice work @vasharma05 just minor cleanups.

**Comment by vasharma05** [comment]
Hi @hadijahkyampeire @denniskigen, I've updated the video. Please re-review the PR.
Thanks!

**Comment by hadijahkyampeire** [comment]
The new fields look great and I see you have corrected the uppercase words, so all looks good.

**Comment by gracepotma** [comment]
Yay, thank you Vineet - what an improvement :)

**Comment by vasharma05** [comment]
Thanks a lot @gracepotma and @hadijahkyampeire !

**Pull request merged**